e2e/fixtures/pages/ExploreProfilesPage.ts

```diff
@@ -87,7 +95,7 @@ export class ExploreProfilesPage extends PyroscopePage {
 
     await this.getProfileTypeSelector().click();
 
-    const menu = this.locator('[role="menu"]');
+    const menu = this.locator('[role="menu"]').last();
```

Comment:
Support multiple open menus containing the same menu item (e.g. when selecting the "goroutine/goroutine" profile type)